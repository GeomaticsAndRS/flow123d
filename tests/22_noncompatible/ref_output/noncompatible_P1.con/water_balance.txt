	"time"	"region"	"quantity"	"flux"	"flux_in"	"flux_out"	"mass"	"source"	"source_in"	"source_out"	"integrated_flux"	"integrated_source"	"error"
	0	"1d"	"water_volume"	0	0	0	0	0	0	0	0	0	0
	0	"2d"	"water_volume"	0	0	0	0	0	0	0	0	0	0
	0	".1d"	"water_volume"	28.0149	0	28.0149	0	0	0	0	0	0	0
	0	".2d_bottom_top"	"water_volume"	-39.3417	-39.3417	0	0	0	0	0	0	0	0
	0	"IMPLICIT BOUNDARY"	"water_volume"	1.27964e-09	-6.34208e-09	7.62172e-09	0	0	0	0	0	0	0
