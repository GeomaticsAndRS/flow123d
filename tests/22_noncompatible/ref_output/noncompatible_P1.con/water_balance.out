# ---------------------------------------- WATER BALANCE ----------------------------------------
# Time of computed water balance: 0.000000


# Boundary water balance:
# [boundary_id] [label]                  [total_balance][total_outflow][total_inflow][time]        
# -----------------------------------------------------------------------------------------------
  4             .1d                        24.4488       24.4488       0             0             
  3             .2d_bottom_top             -36.4701      0             -36.4701      0             
  -3            IMPLICIT BOUNDARY          -1.14618e-10  3.92494e-09   -4.03956e-09  0             
# -----------------------------------------------------------------------------------------------
# total boundary balance                   -12.0212      24.4488       -36.4701      


# Source fluxes over material subdomains:
# [region_id]   [label]                  [total_balance]                            [time]        
# -----------------------------------------------------------------------------------------------
  1             1d                         0                                         0             
  2             2d                         0                                         0             
# -----------------------------------------------------------------------------------------------
# total sources balance                    0             


